ihttp/ihttp.go

```diff
@@ -97,6 +99,18 @@ func (s Handler) Handle(w http.ResponseWriter, req *http.Request) {
 
 		return
 	}
+
+	if len(s.Patch) > len(binary.MagicString) {
```

Comment:
I think this will be overflowable with characters represented in utf8 by > 1 byte.

Also, the handling may read a little better if we enclose everything in a 'if len(s.Patch) > 0' as it should remove the need for else if.

Reply:
Contrary to the naming, both `s.Patch` and `binary.MagicString` are byte slices. I think that should avoid any utf8 weirdness?

Reply:
Yes, that's right.  It sure why I thought Patch was a string.